xtask/src/metrics.rs

```diff
@@ -81,7 +82,11 @@ impl Metrics {
     }
     fn measure_analysis_stats_path(&mut self, name: &str, path: &str) -> Result<()> {
         eprintln!("\nMeasuring analysis-stats/{}", name);
-        let output = cmd!("./target/release/rust-analyzer analysis-stats --quiet {path}").read()?;
+        let output = Command::new("./target/release/rust-analyzer")
+            .args(&["analysis-stats", "--quiet", "--memory-usage", path])
+            .stdout(Stdio::piped())
+            .output()?;
```

Comment:
Wouldn't `output` pipe both stderr and stdout by default? I guess cmd! should have a flag to echo stderr...

Reply:
Ah, right. Yeah, this is to work around `cmd!` not having a matching API.

Reply:
https://github.com/matklad/xshell/pull/11/commits/cf0657944cf0233457b0c150e1c85f5d04f10327